FreeRTOS/Demo/AVR_ATMega323_WinAVR/regtest.c

```diff
@@ -45,312 +45,311 @@ portBASE_TYPE xRegTestError = pdFALSE;
+        asm ( "LDI	r31,	33");
+        asm ( "CPSE	r31,	r30");
+        asm ( "STS	xRegTestError, r0");
+    }
 }
 /*-----------------------------------------------------------*/
 
-static void prvRegisterCheck2( void *pvParameters )
+static void prvRegisterCheck2( void * pvParameters )
 {
-	( void ) pvParameters;
+    ( void ) pvParameters;
 
-	for( ;; )
-	{
-		asm(	"LDI	r31,	1"		);
-		asm( 	"MOV	r0,		r31"	);
-		asm(	"LDI	r31,	2"		);
-		asm( 	"MOV	r1,		r31"	);
-		asm(	"LDI	r31,	3"		);
-		asm( 	"MOV	r2,		r31"	);
-		asm(	"LDI	r31,	4"		);
-		asm( 	"MOV	r3,		r31"	);
-		asm(	"LDI	r31,	5"		);
-		asm( 	"MOV	r4,		r31"	);
-		asm(	"LDI	r31,	6"		);
-		asm( 	"MOV	r5,		r31"	);
-		asm(	"LDI	r31,	7"		);
-		asm( 	"MOV	r6,		r31"	);
-		asm(	"LDI	r31,	8"		);
-		asm( 	"MOV	r7,		r31"	);
-		asm(	"LDI	r31,	9"		);
-		asm( 	"MOV	r8,		r31"	);
-		asm(	"LDI	r31,	10"		);
-		asm( 	"MOV	r9,		r31"	);
-		asm(	"LDI	r31,	11"		);
-		asm( 	"MOV	r10,	r31"	);
-		asm(	"LDI	r31,	12"		);
-		asm( 	"MOV	r11,	r31"	);
-		asm(	"LDI	r31,	13"		);
-		asm( 	"MOV	r12,	r31"	);
-		asm(	"LDI	r31,	14"		);
-		asm( 	"MOV	r13,	r31"	);
-		asm(	"LDI	r31,	15"		);
-		asm( 	"MOV	r14,	r31"	);
-		asm(	"LDI	r31,	16"		);
-		asm( 	"MOV	r15,	r31"	);
-		asm(	"LDI	r16,	17"		);
-		asm(	"LDI	r17,	18"		);
-		asm(	"LDI	r18,	19"		);
-		asm(	"LDI	r19,	20"		);
-		asm(	"LDI	r20,	21"		);
-		asm(	"LDI	r21,	22"		);
-		asm(	"LDI	r22,	23"		);
-		asm(	"LDI	r23,	24"		);
-		asm(	"LDI	r24,	25"		);
-		asm(	"LDI	r25,	26"		);
-		asm(	"LDI	r26,	27"		);
-		asm(	"LDI	r27,	28"		);
-		asm(	"LDI	r30,	29"		);
+    for( ; ; )
+    {
+        asm ( "LDI	r31,	1");
+        asm ( "MOV	r0,		r31");
+        asm ( "LDI	r31,	2");
+        asm ( "MOV	r1,		r31");
+        asm ( "LDI	r31,	3");
+        asm ( "MOV	r2,		r31");
+        asm ( "LDI	r31,	4");
+        asm ( "MOV	r3,		r31");
+        asm ( "LDI	r31,	5");
+        asm ( "MOV	r4,		r31");
+        asm ( "LDI	r31,	6");
+        asm ( "MOV	r5,		r31");
+        asm ( "LDI	r31,	7");
+        asm ( "MOV	r6,		r31");
+        asm ( "LDI	r31,	8");
+        asm ( "MOV	r7,		r31");
+        asm ( "LDI	r31,	9");
+        asm ( "MOV	r8,		r31");
+        asm ( "LDI	r31,	10");
+        asm ( "MOV	r9,		r31");
+        asm ( "LDI	r31,	11");
+        asm ( "MOV	r10,	r31");
+        asm ( "LDI	r31,	12");
+        asm ( "MOV	r11,	r31");
+        asm ( "LDI	r31,	13");
+        asm ( "MOV	r12,	r31");
+        asm ( "LDI	r31,	14");
+        asm ( "MOV	r13,	r31");
+        asm ( "LDI	r31,	15");
+        asm ( "MOV	r14,	r31");
+        asm ( "LDI	r31,	16");
+        asm ( "MOV	r15,	r31");
+        asm ( "LDI	r16,	17");
+        asm ( "LDI	r17,	18");
+        asm ( "LDI	r18,	19");
+        asm ( "LDI	r19,	20");
+        asm ( "LDI	r20,	21");
+        asm ( "LDI	r21,	22");
+        asm ( "LDI	r22,	23");
+        asm ( "LDI	r23,	24");
+        asm ( "LDI	r24,	25");
+        asm ( "LDI	r25,	26");
+        asm ( "LDI	r26,	27");
+        asm ( "LDI	r27,	28");
+        asm ( "LDI	r30,	29");

```

Comment:
Is this space between the asm and the paren character still valid?
Formatting these demo files with uncrustify is a bit tricky, as seen in the [FreeRTOS-Kernel PR #768](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768) to format the portable directory.


I think it might make sense to start by just trimming whitespace and swapping line endings to use `LF` instead of `CLRF`, which I started in #1081 and then follow that up with formatting files that don't contain the `asm(` string?

Reply:
I see the point, and I agree this would cause a problem for the files containing assembly code.
Let me start by raising changes only for the "main" application files and then check the rest.